Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -6616,6 +6632,98 @@ void Document::set_onvisibilitychange(WebIDL::CallbackType* value)
     set_event_handler_attribute(HTML::EventNames::visibilitychange, value);
 }
 
+// https://drafts.csswg.org/css-view-transitions-1/#dom-document-startviewtransition
+// FIXME: Calling document.startViewTransition() without arguments throws TypeError instead of calling this.
+GC::Ptr<ViewTransition::ViewTransition> Document::start_view_transition(Optional<ViewTransition::ViewTransitionUpdateCallback> update_callback)
```

Comment:
do we need to use optional here, considering `ViewTransitionUpdateCallback` is `GC::Ptr` so we could simply check if it's null?